crates/sui-replay/src/lib.rs

```diff
@@ -62,6 +63,23 @@ pub enum ReplayToolCommand {
         path: PathBuf,
     },
 
+    /// Replay transaction
+    #[command(name = "rp")]
+    ProfileTransaction {
+        #[arg(long, short)]
+        tx_digest: String,
+        #[arg(long, short)]
+        show_effects: bool,
+        #[arg(long, short)]
+        diag: bool,
+        #[arg(long, short, allow_hyphen_values = true)]
+        executor_version_override: Option<i64>,
+        #[arg(long, short, allow_hyphen_values = true)]
+        protocol_version_override: Option<i64>,
+        #[arg(long, short, allow_hyphen_values = true)]
+        profile_output_filepath_override: Option<PathBuf>,
```

Comment:
nit: I would drop `override` from the names of these (at least in the actual command-line options), and instead add doc comments explaining what the default is for each of these.

All these command-line options also need doc comments (this is generally true for all the commands though, so I wouldn't be opposed to that being a follow-up as well).

```suggestion
        #[arg(long, short, allow_hyphen_values = true)]
        executor_version: Option<i64>,
        #[arg(long, short, allow_hyphen_values = true)]
        protocol_version: Option<i64>,
        #[arg(long, short, allow_hyphen_values = true)]
        profile_output: Option<PathBuf>,
```